az_quickstart.install

```diff
@@ -226,3 +226,10 @@ function az_quickstart_update_920103() {
     return t('Environment Indicator module installed.');
   }
 }
+
+/**
+ * Install linkit.
+ */
+function az_quickstart_update_920104() {
```

Comment:
Assuming we want to backport this to `2.3.x` we'll want to use `920304` as the update number.  Otherwise `920401`.
```suggestion
function az_quickstart_update_920304() {
```